pages/header/header.js

```diff
@@ -1,13 +1,20 @@
 import MenuDropdown from '../components/MenuDropdown';
 import Logo from '../components/Logo';
 import { CATEGORIES, SORT_ORDER_MAP } from '../../src/constants/products';
-
-export default function Header() {
+export default function Header({ setCategoryFilter, setOrderSort }) {
```

Comment:
Style nitpick:
This should be a lint rule but its preferable to have 1 empty line between imports and code